thinc/util.py

```diff
@@ -50,7 +54,7 @@ def get_array_module(arr):  # pragma: no cover
 
 
 def gpu_is_available():
-    return has_cupy_gpu
+    return has_cupy_gpu or has_torch_mps_gpu
```

Comment:
Why don't we define a `has_gpu` var that incorporates this `or` logic? It would make the check in `prefer_gpu` less awkward, too.

Reply:
Added and updated checks.